composer.json

```diff
@@ -22,7 +22,9 @@
         "json-ld/tests": "@dev"
     },
     "autoload": {
-        "psr-0": { "ML\\JsonLD": "" }
+        "psr-4": { "ML\\JsonLD\\": "src/" }
```

Comment:
What do you mean they are autoloaded? They are just loaded if they are used. Or do you mean that the class map grows? Can't that be worked around with `exclude-from-classmap`?